lib/generators/templates/Capfile

```diff
@@ -0,0 +1,44 @@
+# Load ENV vars via Figaro
+require 'figaro'
+Figaro.application = Figaro::Application.new(environment: 'production', path: File.expand_path('../config/application.yml', __FILE__))
```

Comment:
Why production?

Reply:
the reason behind `production` is that for deployment we only need the production variables values and the global ones.